coral/src/app/features/topics/request/TopicRequest.test.tsx

```diff
@@ -379,74 +433,25 @@ describe("<TopicRequest />", () => {
       });
     });
 
-    // describe("when topic name regex from BE should be applied", () => {
```

Comment:
I forgot to remove this commented out test case in the last PR. We don't validate the regex on our site (at the moment), because the pattern from BE would need to be converted etc.